examples/modify-pin/README.md

```diff
@@ -0,0 +1,60 @@
+# Setting and modifying PIN values
+
+This example shows you the functionality and usage of the following four commands:
+* [set_pin](#set_pin-command) - Sets a new PIN value
+* [change_pin](#change_pin-command) - Changes current PIN value to a new one
+* [unlock_pin](#unlock_pin-command) - Disables PIN using the corresponding PUK
+* [verify_pin](#verify_pin-command) - Verifies a PIN and enables usage of certain command
+
+The contents of `get_reader()` and `activate_card(reader)` have already been covered in the example [get-card-info](../get-card-info). Please reference the previous example if something is unclear in these functions.
+
+## set_pin command
+The `set_pin(reader, pin)` command is used to set a new PIN on the Blockchain Security 2Go card. The PIN value has to have a minimum length of 4 and can not exceed a maximum length of 62 bytes. Example PIN values may look like: `1234`, `abcd`, `1234abcd`, `Even this sentence can work as a PIN!`. Like every PIN you should try to make it secure and unpredictable. After using the command it is highly recommended to always store the returned PUK. This PUK has to be used in case the PIN gets locked (PIN entered incorrectly 3 times). Running the example and selecting the "Set pin" instruction will result in an output similar to:
+
+    Found the specified reader and a Blockchain Security 2Go card!
+    What would you like to do? ("Set pin", "Change pin", "Unlock pin" or "Verify pin")
+    Set pin
+    Please enter a new PIN: 1234
+	PUK to unlock card (hex): dd401ad08a1f0dcd
+
+Note that if the card already has a PIN, then executing this line will return an error. 
```

Comment:
Please add a note that in our implementation the PIN is UTF-8 encoded (e.g. for the pin 1234 the "PIN bytes" 31:32:33:34 will be sent to the card)